src/components/Users/UserForm.tsx

```diff
@@ -92,6 +92,8 @@ export default function UserForm({
       email: z.string().email(t("invalid_email_address")),
       phone_number: validators().phoneNumber.required,
       gender: z.enum(GENDERS, { required_error: t("gender_is_required") }),
+      prefix: z.string().optional(),
+      suffix: z.string().optional(),
```

Comment:
use `z.enum` instead

Reply:
There's gonna be an option for custom prefix/suffix, this is WIP